compiler/test/compilable/test23548.d

```diff
@@ -1,5 +1,4 @@
 // https://issues.dlang.org/show_bug.cgi?id=23548
-// REQUIRED_ARGS: -Icompilable/extra-files/issue23548
-// EXTRA_FILES: extra-files/issue23548/imports/imp23548.d imports/imp23548.c
+// EXTRA_FILES: imports/imp23548.d imports/imp23548.c
```

Comment:
This was a test I introduced to ensure that D modules found on `-I` path do not get overridden by C files from the CWD.

Which this patch, testing this is no longer necessary.  We can just have the D and C files in the same location, and we ensure the D module always matches first.